includes/wc-core-functions.php

```diff
@@ -389,6 +390,7 @@ function get_woocommerce_currency_symbol( $currency = '' ) {
 		'TRY' => '&#8378;',
 		'TWD' => '&#78;&#84;&#36;',
 		'UAH' => '&#8372;',
+		'UGX' => 'UGX',
```

Comment:
Seems like the symbol is `USh`.
Can yon confirm it for me?
https://en.wikipedia.org/wiki/Ugandan_shilling